src/Microsoft.Health.Api/Modules/CorsModule.cs

```diff
@@ -32,7 +32,7 @@ public void Load(IServiceCollection services)
         {
             EnsureArg.IsNotNull(services, nameof(services));
 
-            var corsPolicyBuilder = new CorsPolicyBuilder()
+            CorsPolicyBuilder corsPolicyBuilder = new CorsPolicyBuilder()
```

Comment:
nit: (style) we generally leave it as var when the type is obvious.

Reply:
Oh yes, I know why the analyzer flagged this. It is because we're using method-chaining. If we only called the ctor, it'd like `var`, but with the additional methods it got confused and assumes the type is no longer obvious (although it is).